### PR TITLE
Added tests to verify component-topics mapping for stack analysis

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**
PR to address https://github.com/openshiftio/openshift.io/issues/1458

### CODE REVIEW

**Comment by samuzzal-choudhury** [comment]
[test]

File: integration-tests/features/stack_analyses_v2_minimal.feature [hunk 1]
```diff
@@ -34,6 +34,8 @@ Feature: Stack analysis v2 API Minimal
     Then I should find that alternate components replace user components
     Then I should find the security node for all dependencies
     Then I should find the security node for all alternate components
+    When I wait 60 seconds
+    Then I should find matching topic lists for all user_stack_info/analyzed_dependencies components
```
Comment (tisnik) [inline integration-tests/features/stack_analyses_v2_minimal.feature]: I know that one minute might be 'safe number', but is not it better to perform additional test (in a loop+sleep) that the requested node has been added to the stack analysis?

File: integration-tests/features/steps/stack_analysis.py [hunk 2]
```diff
@@ -634,3 +634,22 @@ def check_stack_analysis_id(context):
...
+    # loop through results for each of the manifest files
+    for result in manifest_results:
+        path = "recommendation/input_stack_topics"
+        input_stack_topics = get_value_using_path(result, path)
+
+        deps = get_value_using_path(result, key)
+
+        for dep in deps:
+            assert len(dep['topic_list']) == len(input_stack_topics[dep['name']])
+            assert sorted(dep['topic_list']) == sorted(input_stack_topics[dep['name']])
```
Comment (tisnik) [inline integration-tests/features/steps/stack_analysis.py]: LGTM, thanks @samuzzal-choudhury

File: integration-tests/features/stack_analyses_v2_minimal.feature (hunk 1 above)
Comment (samuzzal-choudhury) [reply integration-tests/features/stack_analyses_v2_minimal.feature]: @tisnik  I think wait time is not required. I will add another step to verify the prerequisite for this test step.

**Comment by tisnik** [comment]
LGTM, thanks a lot @samuzzal-choudhury . Just let's wait for tests to finish.

**Pull request merged**